src/Components/UserAvatar/index.tsx

```diff
@@ -0,0 +1,39 @@
+import { DefaultAvatar } from "../../assets/Icons";
+
+type AvatarProps = {
+  name?: string;
+  src?: string | undefined;
+  className?: string;
+  onClick?: () => void;
+};
+
+/* To use it =>  <UserAvatar name="Albert John" className="h-32px w-32px" /> */
+/* To use it =>  <UserAvatar src="https://www.kindpng.com/picc/m/236-2362818_anime-sempai-animegirl-heart-kawaii-cute-anime-girl.png" /> */
+
+const UserAvatar = ({ name, src, className, onClick }: AvatarProps) => {
+  const AvatarClasses = "rounded-50px border-2px-white outline flex justify-center";
+  if (name) {
```

Comment:
check the url first and if url is not provided then use name to show user avatar